src/common/utils.cpp

```diff
@@ -93,6 +94,26 @@ std::string join(const std::vector<std::string>& words, const std::string& del /
   return ss.str();
 }
 
+std::vector<std::string> tsv2lists(const std::string& inputText, int inputNum) {
```

Comment:
Please add a comment documenting the function, ideally with an example input and output.